packages/commonwealth/client/scripts/views/components/poll_card.tsx

```diff
@@ -80,17 +81,22 @@ export type CastVoteAttrs = {
   disableVoteButton: boolean;
   timeRemaining: string;
   tooltipErrorMessage: string;
-  onVoteCast: () => any;
+  onVoteCast: (
+    selectedOption: string,
+    handleVoteCast: () => void,
+    isSelected?: boolean
+  ) => any;
```

Comment:
Can we make this return type void? Given we know it doesn't return anything.

Reply:
Done! It typechecks.